docker/Dockerfile.1-13-cpu

```diff
@@ -0,0 +1,27 @@
+ARG TFS_VERSION
+
+FROM tensorflow/serving:${TFS_VERSION} as tfs
+FROM ubuntu:18.04
```

Comment:
I am just trying to be consistent with the official TensorFlow serving repo. I assume this is the version they have tested since this is the official version.